modules/meta.analysis/R/jagify.R

```diff
@@ -33,16 +33,16 @@ jagify <- function(result, use_ghs = TRUE) {
   
   r <- PEcAn.DB::assign.treatments(r)
   r <- PEcAn.utils::summarize.result(r)
-  r <- subset(transform(r, 
-                        stat = as.numeric(stat), 
-                        n = as.numeric(n), 
-                        site_id = as.integer(factor(site_id, unique(site_id))), 
-                        greenhouse = as.integer(factor(greenhouse, unique(greenhouse))),
-                        mean = mean, 
-                        citation_id = citation_id,
-                        ghs = greenhouse,
-                        site = site_id,
-                        trt_name = name), 
+  r <- subset(dplyr::mutate(r, 
+                        stat = as.numeric(.data$stat), 
+                        n = as.numeric(.data$n), 
+                        site_id = as.integer(factor(.data$site_id, unique(.data$site_id))), 
+                        greenhouse = as.integer(factor(.data$greenhouse, unique(.data$greenhouse))),
+                        mean = .data$mean,
+                        citation_id = .data$citation_id,
+                        ghs = .data$greenhouse,
+                        site = .data$site_id,
+                        trt_name = .data$name), 
               select = c("stat", "n", "site_id", "trt_id", "mean", "citation_id", "greenhouse", 
                          "ghs", "treatment_id", "site", "trt_name")) # original versions of greenhouse, treatment_id, site_id, and name
```

Comment:
```suggestion
  r$stat <- as.numeric(r$stat)
  r$n <- as.numeric(r$n)
  r$site_id <- as.integer(factor(r$site_id, unique(r$site_id)))
  r$greenhouse <- as.integer(factor(r$greenhouse, unique(r$greenhouse)))
  r$ghs <- r$greenhouse
  r$site <- r$site_id
  r$trt_name <- r$name
  
  r <- r[, c("stat", "n", "site_id", "trt_id", "mean", "citation_id", "greenhouse", 
             "ghs", "treatment_id", "site", "trt_name")]
```

Reply:
I believe if you do it this way there's no need to declare the `dplyr` import above in DESCRIPTION